extensions/interactions/EndExploration/EndExploration.py

```diff
@@ -64,3 +65,12 @@ class EndExploration(base.BaseInteraction):
         },
         'default_value': [],
     }]
+
+    @classmethod
+    def to_android_end_exploration_proto(cls):
+        """Creates a EndExplorationInstanceDto proto object.
+
+        Returns:
+            EndExplorationInstanceDto. The proto object.
+        """
+        return state_pb2.EndExplorationInstanceDto()
```

Comment:
There's a test in state_domain. 

```py
end_exploration_interaction = state_domain.InteractionInstance(
            'EndExploration',
            {}, [], default_outcome, [], [], None)
        end_exploration_proto = (
            end_exploration_interaction.to_android_interaction_proto())
        self.assertEqual(
            type(end_exploration_proto.end_exploration),
            state_pb2.EndExplorationInstanceDto
        )
```

As we are just testing the type and nothing else to check, I didn't create a particular test file for this.
Let me know if we need, but it will be just duplicate test.

Reply:
I think it's worth having a separate test since the other won't actually fail  if we add more data to this object. You want to verify that it's empty, not just that it's the correct type (for an interaction-specific test; the type check is fine for state_domain since it's essentially delegating verification of the component to the interaction-specific test suite).

Reply:
We cannot directly test if a proto message is empty, not in my understanding. 
Another approach here to test here is to check if the byte size of this proto should be zero.

Reply:
There's usually a way to compare it to its default instance, but the zero byte check is equivalent so this should work.